huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -434,6 +433,70 @@ func resourceEvsVolumeRead(_ context.Context, d *schema.ResourceData, meta inter
 	return nil
 }
 
+func modifyQoS(ctx context.Context, client *golangsdk.ServiceClient, d *schema.ResourceData, cfg config.Config) diag.Diagnostics {
```

Comment:
- Please using the return message.
- Using `schema.TimeoutUpdate` to replace `schema.TimeoutCreate`
- Using `PollInterval` to replace `MinTimeout`.
- Whether judging `vlume_type` is "GPSSD2" or "ESSD2".
- The status `modifying` is not founding in document, please check the status.